aws.go

```diff
@@ -6,17 +6,58 @@ import (
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/cloudwatch"
+	"github.com/percona/rds_exporter/config"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 var (
-	PeriodSeconds = 60
-	DelaySeconds  = 600
-	RangeSeconds  = 600
+	Period = 60 * time.Second
+	Delay  = 600 * time.Second
+	Range  = 600 * time.Second
+
+	latencyDesc = prometheus.NewDesc(
+		"rds_latency",
+		"The difference between the current time and timestamp in the metric itself",
+		[]string{"instance", "region"},
+		map[string]string(nil),
+	)
 )
 
+type Scrape struct {
+	// params
+	instance  config.Instance
+	collector *Collector
```

Comment:
I don't like the idea of having a Collector to create Scrapers, and having Scrapers to have a link back to Collector. This seems overcomplicated to me. Can we do simpler?